docs/DeveloperGuide.md

```diff
@@ -163,13 +164,13 @@ The `Model` component,
 
 ### Storage component
 
-**API** : [`Storage.java`](https://github.com/AY2122S1-CS2103T-W16-3/tp/blob/master/src/main/java/seedu/address/storage/Storage.java)
+**API** : [`Storage.java`](https://github.com/AY2122S1-CS2103T-W16-3/tp/blob/master/src/main/java/seedu/tutoraid/storage/Storage.java)
```

Comment:
Link should no longer contain seedu iirc

Reply:
Yep I have corrected the link, thanks!